maven/spec/dependabot/maven/version_spec.rb

```diff
@@ -69,6 +81,24 @@
 
       it { is_expected.to eq("1.0.0_pre1") }
     end
+
+    context "with a blank version" do
+      let(:version_string) { nil }
```

Comment:
Is there a possibility `version_string` being "" instead of nil? If that is the case what is going to happen?

Reply:
I haven't seen version be an empty string but that is being handled regardless as it is valid input. Anyway, I've added an extra test for it.